src/Microsoft.ML/PredictionModel.cs

```diff
@@ -29,6 +30,39 @@ internal Runtime.EntryPoints.TransformModel PredictorModel
             get { return _predictorModel; }
         }
 
+        /// <summary>
+        /// Returns labels that correspond to indices of the score array in the case of 
+        /// multi-class classification problem.
+        /// </summary>
+        /// <param name="mapping">Label to score mapping</param>
+        /// <param name="scoreColumnName">Name of the score column</param>
+        /// <returns></returns>
+        public bool TryGetScoreLabelMapping(out string[] mapping, string scoreColumnName = DefaultColumnNames.Score)
+        {
+            mapping = null;
+            ISchema schema = _predictorModel.OutputSchema;
+            int colIndex = -1;
+            if (!schema.TryGetColumnIndex(scoreColumnName, out colIndex))
+                return false;
+            
+            int expectedLabelCount = schema.GetColumnType(colIndex).AsVector.ValueCount;
+            if (!schema.HasSlotNames(colIndex, expectedLabelCount))
+                return false;
+
+            VBuffer<DvText> labels = default;
+            schema.GetMetadata(MetadataUtils.Kinds.SlotNames, colIndex, ref labels);
+            VBufferUtils.Densify(ref labels);
```

Comment:
>VBufferUtils.Densify(ref labels); [](start = 12, length = 33)

No no... not `Densify`, `DenseValues`. There is no need for you to materialize a dense array in the sparse case -- `DenseValues` returns an `IEnumerable<DvText>` that you can process.